apps/rpc/src/modules/event/attendance-service.ts

```diff
@@ -300,6 +303,35 @@ export function getAttendanceService(
         )
       }
 
+      if (event.parentId) {
+        if (options.ignoreRegisteredToParent) {
+          logger.info(
```

Comment:
Isnt there intended to be more code here?

Reply:
bruh

Reply:
if it ignores it just logs and moves on